calculator.rb

```diff
@@ -0,0 +1,44 @@
+puts "Welcome to my first calculator. Please be gentle."
+
+puts "What operator would you like to use?"
```

Comment:
I would suggest giving the user a list of options here.

```suggestion
puts "What operator would you like to use?"
puts "Please tell me to add (+), subtract (-), multiply (*), or divide (/)!"
```